thunder/common.py

```diff
@@ -672,10 +671,6 @@ def _execute_trace(
     # Constructs the Python callable
     c = extrace.python_callable()
 
-    # TODO RC1 Mark this option as experimental
-    if compile_data.use_cudagraphs:
-        c = CUDAGraphExecutor(c, num_constant_args=compile_data.num_constant_args)
-
```

Comment:
@IvanYashchuk , from https://github.com/Lightning-AI/lightning-thunder/pull/595 I see that, ultimately, a plan is to remove `_execute_trace` as well?

Reply:
Yes, it's used only in `thunder.compile`.